test/sharness/lib/test-lib-hashes.sh

```diff
@@ -1,6 +1,6 @@
 # this file defines several useful hashes used across the test codebase.
 # thus they can be defined + changed in one place
 
-HASH_WELCOME_DOCS="Qmcqtw8FfrVSBaRmbWwHxt3AuySBhJLcvmFYi3Lbc4xnwj"
+HASH_WELCOME_DOCS="QmUaZRPqxSpkeQ5QvtYWYzRQA9K4zDPfQkkYjVgLx2XWrd"
```

Comment:
The hash changed because of a fixed typo (6db1abbe0d21eac3dda5ed9219b3f11e77435a7d) that never made it into the binary.